pkg/integrations/blackbox_exporter/blackbox_exporter.go

```diff
@@ -39,18 +41,24 @@ type BlackboxTarget struct {
 
 // Config configures the Blackbox integration.
 type Config struct {
-	BlackboxConfigFile string                 `yaml:"config_file,omitempty"`
-	BlackboxTargets    []BlackboxTarget       `yaml:"blackbox_targets"`
-	BlackboxConfig     blackbox_config.Config `yaml:"blackbox_config,omitempty"`
-	ProbeTimeoutOffset float64                `yaml:"probe_timeout_offset,omitempty"`
+	BlackboxConfigFile string           `yaml:"config_file,omitempty"`
+	BlackboxTargets    []BlackboxTarget `yaml:"blackbox_targets"`
+	BlackboxConfig     util.RawYAML     `yaml:"blackbox_config,omitempty"`
```

Comment:
As an aside:

This would normally be the kind of thing which would be a breaking change to the code API. To me, that's re-enforcing the idea of putting everything in internal/ for 1.0 until we're ready to pick a subset of packages to expose.